lib/ads/client_side_ad_manager.js

```diff
@@ -97,6 +97,40 @@ shaka.ads.ClientSideAdManager = class {
     this.adsLoader_.requestAds(imaRequest);
   }
 
+  /**
+   * Simple call to IMA's start() method
+   */
+  startAdManager(imaAdsManager) {
+    if (imaAdsManager && imaAdsManager.start) {
+      imaAdsManager.start();
+    }
+  }
+
+  /**
+   * Start ads after checking autoplay
+   */
+  startAds(imaAdsManager) {
```

Comment:
The parameter is undocumented and comes from `this`.  You should just drop the parameter and refer to `this.imaAdsManager_`.

If you need to keep the value constant around the async part of the code, capture it in a local variable below.

You should also make startAds private (ends with underscore, has `@private` compiler annotation).